client/src/app/shared/services/ingest.service.spec.ts

```diff
@@ -83,4 +83,35 @@ describe('Ingest Service', () => {
 
     ['files', 'protocols', 'biomaterials', 'processes', 'projects'].forEach(makeTest);
   });
+
+  describe('standard CRUD helpers', () => {
+    it('should get the given url', () => {
+      const url = '42.com/meaning/life';
+      service.get(url).subscribe(() => {});
+      const req = httpTestingController.expectOne(url);
+      expect(req.request.method).toEqual('GET');
+    });
+
+    it('should send a PUT request to the given url', () => {
+      const url = '42.com/meaning/life';
+      const body = {
+        name: 'Zaphod'
+      };
+      service.put(url, body).subscribe(() => {});
+      const req = httpTestingController.expectOne(url);
+      expect(req.request.method).toEqual('PUT');
+      expect(req.request.body).toEqual(body);
+    });
+
+    it('should send a PATCH request to the given url', () => {
+      const url = '42.com/meaning/life';
+      const body = {
+        name: 'Zaphod'
+      };
+      service.patch(url, body).subscribe(() => {});
+      const req = httpTestingController.expectOne(url);
+      expect(req.request.method).toEqual('PATCH');
+      expect(req.request.body).toEqual(body);
+    });
+  });
```

Comment:
i wonder what these tests are for

Reply:
Yeah, sorry. I added a comment in the PR but I think I must have branch off of my unit tests branch instead of master. These are unrelated but are just testing basic ingest service functions